package.json

```diff
@@ -71,13 +77,15 @@
     "eslint-config-airbnb-base": "14.2.1",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-mocha": "9.0.0",
+    "fs-extra": "^10.0.0",
     "husky": "4.3.8",
     "jest": "^27.4.7",
     "jest-junit": "^13.0.0",
     "less": "^4.1.2",
     "lint-staged": "11.2.6",
     "mocha-junit-reporter": "2.0.2",
     "shx": "0.3.4",
+    "yargs": "^17.2.1",
```

Comment:
Should be looked to version `17.1.1`. Don't forget to run `yarn` after so that the `yarn.lock` file gets updated.